tests/accessor/accessor_api_local_common.h

```diff
@@ -62,6 +62,24 @@ class check_local_accessor_api_methods {
   size_t count;
   size_t size;
```

Comment:
Are these 2 members used somewhere?

Reply:
Yes, when checking `get_count` and `get_size` later on